bcolz/ctable.py

```diff
@@ -371,7 +371,10 @@ def append(self, cols):
             if sclist and not hasattr(column, '__len__'):
```

Comment:
This code is a bit convoluted, can you add ``or isintance(column, _strtypes)`` to this line, rather than the many lines below, or alternatively, use:

```
clen = len(column) if not isinstance(column, _strtypes) else 1
```
or:

```
clen = 1 if isinstance(column, _strtypes) else len(column)
```